ethminer/MinerAux.h

```diff
@@ -1047,6 +1066,7 @@ class MinerCLI
 	unsigned m_openclPlatform = 0;
 	unsigned m_miningThreads = UINT_MAX;
 	bool m_shouldListDevices = false;
+	unsigned m_selectedKernel = 0;  ///< A generic value containing a kernel enum. Different between OCL and CUDA.
```

Comment:
As said before would not use the same variable for ocl + cuda. since you can use ``-X`` for both if you have mixed cards.

Reply:
Fair enough, I didn't think of mixed systems.

Reply:
yes, that would be premium :)